forms/HeaderField.php

```diff
@@ -17,31 +17,17 @@ class HeaderField extends DatalessField {
 	 */
 	protected $headingLevel = 2;
 
+	protected $schemaDataType = self::SCHEMA_DATA_TYPE_STRUCTURAL;
+
+	protected $schemaComponent = 'HeaderField';
+
 	/**
 	 * @param string $name
-	 * @param null|string $title
+	 * @param string $title
 	 * @param int $headingLevel
 	 */
-	public function __construct($name, $title = null, $headingLevel = 2) {
```

Comment:
Has the `new HeaderField($title, $headlingLevel)` (rather than `new HeaderField($name, $title, $headlingLevel)`) been deprecated? That'll break heaps of existing use cases, from my experience with SilverStripe project code.